### PR TITLE
Add a howto for community reviews using Reviewable.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/toktok/website/36)

### CODE REVIEW

**Comment by fcore117** [comment]
Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[.travis.yml, line 19 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEqJQhe77Aan3FEGgf:-KaEqJQiBccAcvKk3p1X:b-wn207b) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/.travis.yml#L19)):*
> ```YAML
> 
> install:
> - gem install jekyll mdl guard-livereload
> ```

Does Travis really need live reloading of the page? No code committed should even have that script uncommented.

---

*[toktok/reviews.md, line 10 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaElj2QjK3kZtnNyP3W:-KaElj2QjK3kZtnNyP3X:b2gv278) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L10)):*
> ```Markdown
> The TokTok review process requires at least three reviews for every pull
> request. One of the reviews must be from a developer. The two other reviews
> can be from the developer but can also be from the community. The idea is that
> ```

"a developer,"

---

*[toktok/reviews.md, line 31 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEmlAEJK1kVs33JNsX:-KaEmlAEJK1kVs33JNsY:bunhfgg) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L31)):*
> ```Markdown
> # Step 2: Log in to Reviewable
> 
> -   Open the Reviewable link in the "#" column on the page or on the PR
> ```

"or on GitHub in the PR description itself" maybe?

---

*[toktok/reviews.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEnAfBxOzdgfcbv8vK:-KaEnAfCFBRQWtonWLze:b-keqlgt) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L50)):*
> ```Markdown
> -   Now comes your review: your task is now to check whether the change makes
>     sense. In this example, we see that `assoc4` was checked in an IPv6 code
>     block. Now that is fixed. Seems legit :).
> ```

"Seems legit. :)"?

---

*[toktok/_layouts/default.html, line 16 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEoCUZQ0ccjIM26nOH:-KaEoCUZQ0ccjIM26nOI:b-i3s3eb) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/_layouts/default.html#L16)):*
> ```HTML
>     <script>const relative = "{{ relative }}";</script>
>     {% endif %}
>

**Comment by iphydf** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[.travis.yml, line 19 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEqJQhe77Aan3FEGgf:-KaEzMpwGIKxQxBjM6Jv:bk8s6d6) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/.travis.yml#L19)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

Does Travis really need live reloading of the page? No code committed should even have that script uncommented.
</blockquote></details>

It doesn't, but it's easier if you only need to add uncomment the javascript link when developing the page. If I remove this, I need to comment out the references in the Gemfile as well. It doesn't seem to add much time (still about 1 minute 50 seconds like some other page builds). Should I remove it altogether and forget about live reloading? I found it to be quite useful while writing the page.

---

*[toktok/reviews.md, line 10 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaElj2QjK3kZtnNyP3W:-KaF-3XnFpV1_lNpSjAU:b-896fix) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L10)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

"a developer,"
</blockquote></details>

Done.

---

*[toktok/reviews.md, line 31 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEmlAEJK1kVs33JNsX:-KaF-91rOFIL6rsMHPKL:b-896fix) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L31)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

"or on GitHub in the PR description itself" maybe?
</blockquote></details>

Done.

---

*[toktok/reviews.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEnAfBxOzdgfcbv8vK:-KaF-AEN9eoyG3kmUZ-i:b-ojcjh4) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L50)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

"Seems legit. :)"?
</blockquote></details>

*Shrug* poor attempt at being funny. https://cdn.meme.am/cache/instances/folder268/60114268.jpg

---

*[toktok/_layouts/default.html, line 16 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEoCUZQ0ccjIM26nOH:-KaF-bsEEEvgipGsz8da:b-896fix) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/_layouts/default.html#L16)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

Put these comments in the Liquid blocks so they're not present on the rendered website.
`{% comment %} Comment not present in production. :D {% endcomment %}`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
Still missing step1.png

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[.travis.yml, line 19 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEqJQhe77Aan3FEGgf:-KaFGao_cGHnE5ZOiD-g:by88n4b) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/.travis.yml#L19)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

It doesn't, but it's easier if you only need to add uncomment the javascript link when developing the page. If I remove this, I need to comment out the references in the Gemfile as well. It doesn't seem to add much time (still about 1 minute 50 seconds like some other page builds). Should I remove it altogether and forget about live reloading? I found it to be quite useful while writing the page.
</blockquote></details>

I didn't realise. Leave it in. :)

---

*[toktok/reviews.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEnAfBxOzdgfcbv8vK:-KaFGnphFXrZdgNFfUVg:b-q0nsu1) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L50)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

*Shrug* poor attempt at being funny. https://cdn.meme.am/cache/instances/folder268/60114268.jpg
</blockquote></details>

Noo. I moved the full stop. It was funny. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36#-:-KaFG_UyUch8-YiqRt1w:b-acpal3)*

**Comment by iphydf** [comment]
Should I put a screenshot of the pulls page? Seemed a bit redundant since users can just click on it, and it's on the same site.

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36#-:-KaFIEm_rJ3AhcjpAN-a:brjdoxo)*

**Comment by iphydf** [comment]
Review status: 10 of 11 files reviewed at latest revision, 1 unresolved discussion.

---

*[toktok/reviews.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/36#-KaEnAfBxOzdgfcbv8vK:-KaFIbF4qP09IWMdPf7k:b-k053km) ([raw file](https://github.com/toktok/website/blob/85d808dc184797ef556b9bb9f76b5fc9ebc98a28/toktok/reviews.md#L50)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

Noo. I moved the full stop. It was funny. 
</blockquote></details>

Ahh, misunderstood :) restored and changed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[toktok/reviews.md, line 40 at r3](https://reviewable.io:443/reviews/toktok/website/36#-KaFJI2tzb6BoDMgxxdJ:-KaFJI2ueOUnsj4BiAsy:b-ni599p) ([raw file](https://github.com/toktok/website/blob/cd2970a34634506cc99bd091800c075a93c32c16/toktok/reviews.md#L40)):*
> ```Markdown
>     click this button, you should be able to see the changes under review.
> 
> Step 1
> ```

I guess I should have made the comment about the missing image "step1.png" here. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by iphydf** [comment]
Review status: 10 of 11 files reviewed at latest revision, 1 unresolved discussion.

---

*[toktok/reviews.md, line 40 at r3](https://reviewable.io:443/reviews/toktok/website/36#-KaFJI2tzb6BoDMgxxdJ:-KaFKUQF_gzBruKwO-TQ:b-9jcf7x) ([raw file](https://github.com/toktok/website/blob/cd2970a34634506cc99bd091800c075a93c32c16/toktok/reviews.md#L40)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

I guess I should have made the comment about the missing image "step1.png" here. 
</blockquote></details>

Oops. Done :)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36#-:-KaFLCdV2TU-YAjsDxTx:b-jbsf8z)*

**Comment by iphydf** [comment]
This needs another review after rebase.

File: toktok/reviews.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+-   Scroll down to see the file changes. You will find something like the
+    screenshot below. Here, we see that someone changed a `4` into a `6` in a
+    file named `DHT.c`. The PR description explains why this was done: "Fix
+    logging condition for IPv6 client timestamp updates".
+-   Now comes your review: your task is now to check whether the change makes
+    sense. In this example, we see that `assoc4` was checked in an IPv6 code
+    block. Now that is fixed. Seems legit. :)
+-   If you don't understand what this change does, ask a question. We will be
+    happy to explain what we're doing. This helps not only you, but also
+    anyone in the future who may be digging through the commit history. It
```
Comment (GrayHatter) [inline toktok/reviews.md:53]: It's also helps developer catch mistakes. I've explained a simple change to someone once, and caught a nearby error that I missed the first go around.

File: toktok/reviews.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    format.
+
+![Step 3](static/img/reviews/step3.png)
+
+# Step 4
+
+When you are done with a file, click the eye symbol in the corner. This marks
+the file as reviewed. Even if you have put comments on it, still mark it as
+reviewed, which just means that you saw it. It does not mean that you consider
+the file OK. Your comments need to be explicitly marked resolved by you
```
Comment (GrayHatter) [inline toktok/reviews.md:71]: If for any reason you're not 100% sure about the file, leave a comment. That way even though you've seen the file, (according to reviewable) everyone knows why, or what you're waiting on.

**Comment by GrayHatter** [comment]
Reviewed 7 of 11 files at r1, 1 of 2 files at r2, 1 of 1 files at r4, 2 of 2 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
Reviewed 2 of 2 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by iphydf** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[toktok/reviews.md, line 53 at r5](https://reviewable.io:443/reviews/toktok/website/36#-KaDSj9S0HD6_eEfzDUM-r5-53:-KaHKL4-etWN283VGSQ0:b-896fix) ([raw file](https://github.com/toktok/website/blob/672c5c470eca056638371d41f1c7bb9788215441/toktok/reviews.md#L53)):*
<details><summary><i>Previously, GrayHatter (Gregory Mullen) wrote…</i></summary><blockquote>

It's also helps developer catch mistakes. I've explained a simple change to someone once, and caught a nearby error that I missed the first go around.
</blockquote></details>

Done.

---

*[toktok/reviews.md, line 69 at r5](https://reviewable.io:443/reviews/toktok/website/36#-KaDSj9S0HD6_eEfzDUM-r5-69:-KaHMrMjESBtce9wbQK8:b7lc6ly) ([raw file](https://github.com/toktok/website/blob/672c5c470eca056638371d41f1c7bb9788215441/toktok/reviews.md#L69)):*
<details><summary><i>Previously, GrayHatter (Gregory Mullen) wrote…</i></summary><blockquote>

If for any reason you're not 100% sure about the file, leave a comment. That way even though you've seen the file, (according to reviewable) everyone knows why, or what you're waiting on.
</blockquote></details>

I've thought about it for a while, but I still don't understand what information this adds. Can you rephrase this?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by fcore117** [comment]
Reviewed 1 of 2 files at r2, 2 of 2 files at r5, 1 of 1 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by robinlinden** [comment]
Reviewed 1 of 1 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by GrayHatter** [comment]
:lgtm: 

---

Reviewed 1 of 1 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[toktok/reviews.md, line 53 at r5](https://reviewable.io:443/reviews/toktok/website/36#-KaDSj9S0HD6_eEfzDUM-r5-53:-KaLlN8i5E0mpsuvFv4v:b-1sxit0) ([raw file](https://github.com/toktok/website/blob/672c5c470eca056638371d41f1c7bb9788215441/toktok/reviews.md#L53)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

Done.
</blockquote></details>

LGTM, that's exactly what I was thinking too. 

---

*[toktok/reviews.md, line 69 at r5](https://reviewable.io:443/reviews/toktok/website/36#-KaDSj9S0HD6_eEfzDUM-r5-69:-KaLlQylnM1k-hD0UpBI:b-r8epms) ([raw file](https://github.com/toktok/website/blob/672c5c470eca056638371d41f1c7bb9788215441/toktok/reviews.md#L69)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

I've thought about it for a while, but I still don't understand what information this adds. Can you rephrase this?
</blockquote></details>

I don't want reviewers to "green light" a file they want to ask questions on, and then not ask a question. 

My worry is this language might be interpreted as, "I've seen this file, it's has major problems, but I've seen it." Because if the user doesn't leave a comment, then we have a false positive for number of reviewers for files.

Maybe reword this section to be fit more of suggested order of operations, instead of a description of how things work?

If I'm still unclear, ping me on IRC and I'll offer a reword that I think fixes the problem I have with this section.

Either way, I'm currently not blocking so I'm fine if you merge with this outstanding.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36#-:-KaLmNTR_26vpOKEr3L6:bg7igkv)*

**Comment by robinlinden** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[toktok/reviews.md, line 89 at r6](https://reviewable.io:443/reviews/toktok/website/36#-KaNWgGLStn7d5mH3e0D:-KaNWgGLStn7d5mH3e0E:b-5awmpz) ([raw file](https://github.com/toktok/website/blob/2ed1bb3b0b4901fc9cad55860ad8c6a4312a83b9/toktok/reviews.md#L89)):*
> ```Markdown
> first one.
> 
> Step 5
> ```

Turns out this file is missing too. :x Want to add it or should I open a PR deleting this line?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/36)*

**Comment by fcore117** [comment]
can this pull canceled then edited so step5.png works?
